### PR TITLE
Catch correct warning [WIP]

### PR DESCRIPTION
**Pull request by kevinburke**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,7 @@
 language: python
 python:
-  - "2.6"
```
Comment (shazow) [inline .travis.yml:3]: Why these changes?

Comment (kevinburke) [inline .travis.yml:3]: ResourceWarning is only raised on Python3, was just trying to run the tests
I cared about to shorten the feedback loop (I can't repro this locally).

In any event the code doesn't work, I am working on it.

## 

Kevin Burke
phone: 925.271.7005 | twentymilliseconds.com

**Comment by shazow** [comment]
K, added [WIP] to title. Ping me when it's ready for review. :)
